.github/pull_request_template.md

```diff
@@ -9,3 +19,10 @@ _How did you solve the problem?_
 
 
```

Comment:
Wow I really like this! If we can all adhere to this very closely, our project will be S O L I D